### ISSUE TITLE
[New install] error: cannot connect to socket at '/nix/var/nix/daemon-socket/socket': Connection refused

### ISSUE THREAD

**Issue by ahmedelgabri** [LnL7/nix-darwin#610]
My nix installation (_MBP mid 2015 running macOS Monterey `12.6.3`_) ended up being in a bad state somehow, so I followed the nix uninstall steps https://nixos.org/manual/nix/stable/installation/installing-binary.html#macos and installed nix from scratch. So far so good, after installing `nix-darwin` & running `darwin-rebuild switch --flake .`

I got an error about this

```
setting up /etc...
error: not linking environment.etc."nix/nix.conf" because /etc/nix/nix.conf already exists, skipping...
existing file has unknown content ff08c12813680da98c4240328f828647b67a65ba7aa89c022bd8072cba862cf1, move and activate again to apply
```

I looked around and saw that moving a file to a backup file could work, I did this

```
sudo mv /etc/nix/nix.conf /etc/nix/nix.conf.bkp
```

I ran `darwin-rebuild` again but this time it failed on this missing `org.nixos.nix-daemon` which it couldn't create https://github.com/NixOS/nix/issues/2780#issuecomment-1450853871 

After this, whenever I try any command I get the following error

```
error: cannot connect to socket at '/nix/var/nix/daemon-socket/socket': Connection refused
```

**Comment by andrewcrook** [comment]
@ahmedelgabri  did you get round this?  
I have the same issue, darwin-rebuild failed 
```
setting up /etc...
error: not linking environment.etc."nix/nix.conf" because /etc/nix/nix.conf already exists, skipping...
existing file has unknown content ff08c12813680da98c4240328f828647b67a65ba7aa89c022bd8072cba862cf1, move and activate again to apply
```
I then get 
`error: cannot connect to socket at '/nix/var/nix/daemon-socket/socket': Connection refused`

Then when I shutdown and come back to try and resolve the issue I find that none of the nix or darwin commands are available.

**Comment by ahmedelgabri** [comment]
@andrewcrook I had to re-install nix but this time I used https://zero-to-nix.com/concepts/nix-installer, after that everything worked fine.

**Comment by jeff-hykin** [comment]
and, just a heads up, unlike a normal reinstall, which involves fully uninstalling and nuking my /nix (which is around 100Gb)
The nix-concepts installer just recognized my existing (broken) install and fixed it it 30sec while preserving everything.
(thanks for linking that @ahmedelgabri)

TLDR; This is what I ran
```
curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install
```

**Comment by Sleepful** [comment]
no idea why the deameon was down, this brought it back on an official nix installation (no detsys)
```
     sudo launchctl bootstrap system /Library/LaunchDaemons/org.nixos.nix-daemon.plist 2>/dev/null || true
     sudo launchctl kickstart -k system/org.nixos.nix-daemon

```
